fe/src/main/java/org/apache/doris/analysis/CreateMaterializedViewStmt.java

```diff
@@ -50,7 +53,21 @@
  * [PROPERTIES ("key" = "value")]
  */
 public class CreateMaterializedViewStmt extends DdlStmt {
-    public static final String MATERIALIZED_VIEW_NAME_PRFIX = "__doris_materialized_view_";
+    public static final String MATERIALIZED_VIEW_NAME_PREFIX = "mv_";
+    public static final Map<String, MVColumnPattern> FN_NAME_TO_PATTERN;
+
+    static {
+        FN_NAME_TO_PATTERN = Maps.newHashMap();
+        FN_NAME_TO_PATTERN.put(AggregateType.SUM.name().toLowerCase(),
+                new MVColumnOneChildPattern(AggregateType.SUM.name().toLowerCase()));
+        FN_NAME_TO_PATTERN.put(AggregateType.MIN.name().toLowerCase(),
+                new MVColumnOneChildPattern(AggregateType.MIN.name().toLowerCase()));
+        FN_NAME_TO_PATTERN.put(AggregateType.MAX.name().toLowerCase(),
+                new MVColumnOneChildPattern(AggregateType.MAX.name().toLowerCase()));
+        FN_NAME_TO_PATTERN.put("count", new MVColumnOneChildPattern("count"));
+        FN_NAME_TO_PATTERN.put("bitmap_union", new MVColumnBitmapUnionPattern());
+        FN_NAME_TO_PATTERN.put("hll_union", new MVColumnHLLUnionPattern());
+    }
```

Comment:
Can you add comment which pattern is suitable for AggregateModel, which pattern is suitable for DuplicatedModel? if it is a need to add a type to distinguish the  case.

Reply:
The patterns in here do not distinguish Agg model and duplicate model. These patterns are compatible.